modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -73,9 +73,7 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 
 ** For installations on FCP-type disks, complete the following tasks:
 ... Use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. For multipathing repeat this step for each additional path.
```

Comment:
can we get a not below, that states: 

"When installing with multiple paths, multipathing needs to be enabled after the installation, not at a later point in time, as this can cause problems."